app/models/manageiq/providers/hawkular/datawarehouse_manager.rb

```diff
@@ -38,38 +39,42 @@ def self.verify_ssl_mode
     end
 
     # Hawkular Client
-    def self.raw_connect(hostname, port, token, type = :alerts)
+    def self.raw_connect(options)
+      type = options[:type] || :alerts
```

Comment:
In all other providers, this key is called `:service`.  Does this *have* to be `:type`?  Or, can it be `:service` instead?

---

EDIT
I just realized that the `type` was a preexisting method param.  I won't hold up this PR on my comment here.  @lucasponce can you take a note here and switch this to `:service` in a separate PR?